jobs/upstream.go

```diff
@@ -43,8 +42,19 @@ func SyncWithUpstream(ctx api.Context) error {
 	return nil
 }
 
+func reconcileTable(ctx api.Context, table string) error {
+	newCtx, span := tracer.Start(ctx, fmt.Sprintf("reconcile-%s", table))
+	defer span.End()
+
+	ctx = ctx.WithContext(newCtx)
+	reconciler := upstream.NewUpstreamReconciler(api.UpstreamConf, ReconcilePageSize)
+
+	return reconciler.SyncAfter(ctx, table, ReconcileMaxAge)
+}
+
```

Comment:
Added a new span for each table we're reconciling because I think it gives a better visual to the trace

![image](https://github.com/flanksource/incident-commander/assets/13419481/89e44aad-71e0-4b1c-be51-d28f2200c54a)

Reply:
vs 
![image](https://github.com/flanksource/incident-commander/assets/13419481/005073c0-77af-4778-8c3f-a7cd1c5dd6a0)